src/http-middlewares/before/add-query-params.js

```diff
@@ -2,27 +2,29 @@
 
 const querystring = require('querystring');
 
+const isToken = /{{.*?}}/g;
```

Comment:
Maybe `isCurlies` is a better name?

Reply:
I know they are referred to as `tokens` in the frontend and generically throughout Formatic for any things to be replaced by some other value.

I think `isCurlies` does technically describe the regEx. But the regEx is pretty simple, and `isCurlies` doesn't really describe what we're checking for. What about `shouldBeReplaced`? For me, `isCurlies` would convey no more extra context than just using the regEx itself: `/{{.*?}}/g.test(value)`. But I'm not stuck on it either.

Reply:
Thanks for clarifying! "curlies" is what we call them in the backend. But yeah, it's only a short regex so I don't see it getting confused either way, so it's your call. 👍

Reply:
Ah ok! If they're "curlies" on the backend, then "curlies" they shall be 😄 I'd rather stick to the same names we're using. Thanks @eliangcs!!